amgut/test/test_dbpatch_source.py

```diff
@@ -0,0 +1,104 @@
+from unittest import TestCase, main
```

Comment:
I'd like to keep it in, because it should that no all tests did correctly revert DB changes.